src/libs/OptionsListUtils.ts

```diff
@@ -1837,12 +1836,8 @@ function getShareDestinationOptions(
  * @param member - personalDetails or userToInvite
  * @param config - keys to overwrite the default values
  */
-function formatMemberForList(member: ReportUtils.OptionData, config: ReportUtils.OptionData | EmptyObject = {}): MemberForList | undefined {
```

Comment:
have we checked all usages to see that it's okay to remove `config`?

Reply:
Double checked if everything works. This method is used only in two places and none of them use `config` param.